contrib/babelfishpg_tsql/sql/sys_views.sql

```diff
@@ -2594,6 +2594,137 @@ SELECT
 WHERE FALSE;
 GRANT SELECT ON sys.spatial_index_tessellations TO PUBLIC;
 
+
+-- This helper function is used when getting the max length from a value in the typmod array in pg_proc.probin
+CREATE OR REPLACE FUNCTION sys.tsql_typmod_array_max_length_helper(IN type TEXT, IN typelen INT, IN typemod INT, IN for_sys_types boolean DEFAULT false) 
+RETURNS smallint 
+AS $$
+DECLARE
+v_type TEXT COLLATE "C" := type;
+BEGIN
+	
+	IF v_type = 'sysname' THEN
+		RETURN 256;
+	ELSIF (v_type in ('char', 'bpchar', 'varchar', 'binary', 'varbinary', 'nchar', 'nvarchar', 'sysname'))
```

Comment:
`sysname` was already checked in the above.

Reply:
Ack, will remove the sysname in the `else if` statement.